.env.example

```diff
@@ -0,0 +1 @@
+env=staging
```

Comment:
let's remove rename this variable to `CONTXT_ENV` (so that's it's namespaced to our cli) and remove all the  `dotenv` calls as i don't think that would be expected behavior (i.e. when you run `docker ls` you don't expect docker to be loading a `.env` file in your current directory). 

a cli user will have to explicitly set the var, like `CONTXT_ENV=staging contxt get ...`. note, i think a flag would be preferable because it's more discoverable but this is fine to get it working (i.e. `contxt --env=staging ...`)